2-Authorization-I/1-call-graph/README.md

```diff
@@ -130,15 +130,10 @@ To manually register the apps, as a first step you'll need to:
 1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
 1. Since this app signs-in users, we will now proceed to select **delegated permissions**, which is is required by apps signing-in users.
```

Comment:
Steps for adding a redirect uri is missing
Let's also use a redirect uri such as localhost:4200/auth, and show how to handle the response there using MsalRedirectComponent